app/inputs/rich_text_input.rb

```diff
@@ -1,6 +1,5 @@
 class RichTextInput < SimpleForm::Inputs::TextInput
-  def input
-    input_html_options[:class] += [:wysihtml5] 
-    @builder.text_area(attribute_name, input_html_options)
+  def input(wrapper_options = nil)
```

Comment:
If this parameter is necessary, you could add an underscore to the front of the parameter name to signal that you are going to throw away the value.